lib/modes/ingest.ex

```diff
@@ -4,52 +4,37 @@ defmodule SonicClient.Modes.Ingest do
   @default_bucket_name "default_bucket"
 
   def push(conn, collection, object, term) do
-    TcpConnection.send(
-      conn,
-      "PUSH #{collection} #{@default_bucket_name} #{object} \"#{term}\""
-    )
+    command = "PUSH #{collection} #{@default_bucket_name} #{object} \"#{term}\""
```

Comment:
I'm not sure if you know it but we can use sigils here like: 
`~s(PUSH #{collection} #{@default_bucket_name} #{object} "#{term}")`
https://elixir-lang.org/getting-started/sigils.html#strings

Reply:
Nope, did not know 🙂